core/src/bin/test_oidc_call_back.rs

```diff
@@ -0,0 +1,28 @@
+use mongo_odbc_core::oidc_auth::*;
+
+#[tokio::main]
+async fn main() {
+    let c = CallbackContext {
+        idp_info: Some(IdpServerInfo {
+            issuer: "https://mongodb-dev.okta.com/oauth2/ausqrxbcr53xakaRR357".to_string(),
+            client_id: Some("0oarvap2r7PmNIBsS357".to_string()),
+            // show that we get a refresh_token even when we don't ask for it here
+            request_scopes: Some(vec!["openid".to_string()]),
+        }),
+        refresh_token: None,
+        timeout_seconds: None,
+        version: 1,
+    };
+    let mut refresh_c = c.clone();
+    let IdpServerResponse {
+        access_token: _,
+        expires: _,
+        refresh_token,
```

Comment:
this is similar to the previous refresh test, but it uses oidc_call_back in both places.